src/dispatching.rs

```diff
@@ -58,25 +58,29 @@
 //! # async fn receive_product_selection() -> HandlerResult { Ok(()) }
 //! #
 //! fn schema() -> UpdateHandler<Box<dyn std::error::Error + Send + Sync + 'static>> {
-//!     let command_handler = teloxide::filter_command::<Command, _>()
+//!     use teloxide::dispatching::dialogue;
+//!     use teloxide::filter_command;
+//!     use dptree::case;
```

Comment:
This looks quite ugly, I think we should lift these imports to top-level.

Reply:
By the way, I think we should only import `dialogue` and `filter_command`, otherwise `case![...]` looks strange without the `dptree::` prefix, at least in the example.